### PR TITLE
Coursera formatting updates

### PR DESCRIPTION
**Pull request by ehumph**
After the beta testing on Coursera, we are making some formatting updates to respond to some negative feedback from reviewers. 

First, we are renaming the course to "Efficient Programming with AI" and making it clearer what the scope of this course is.

Second, we are clarifying that our writing is all original work and that AI was used for editing, changing tone, and planning. 

Third, the activities are now separate chapters to make it more obvious that they should be _tried_, not just read.

### CODE REVIEW

File: 04-refactoring_activity.Rmd [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+```{r, include = FALSE}
+# enable python code previews; must use python 3
+library(reticulate)
+use_python("/usr/bin/python3")
+
+ottrpal::set_knitr_image_path()
+```
+
+
+# Refactoring Code: Hands-On Exercise
```
Comment (cansavvy) [inline 04-refactoring_activity.Rmd:11]: You could have been even lazier and just changed the H2's to H1's and that would have made a new chapter in bookdown and for the resulting OTTR stuff. But this is probably better.

File: _bookdown.yml [hunk 2]
```diff
@@ -5,11 +5,15 @@ rmd_files: ["index.Rmd",
             "01-intro.Rmd",
             "02-ethics.Rmd",
             "03-writing_code.Rmd",
+            "03-writing_code_activity.Rmd",
             "04-refactoring.Rmd",
+            "04-refactoring_activity.Rmd",
             "05-annotation.Rmd",
```
Comment (cansavvy) [inline _bookdown.yml:11]: I have annotation activities I've used in workshops, I should probably add one here. But I guess that's for another day. I also should change it so it uses Phind. But also probably for another day.

File: index.Rmd [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "AI for Software Development"
+title: "Efficient Programming with AI"
```
Comment (cansavvy) [inline index.Rmd:2]: Noice.

**Review by cansavvy — APPROVED** [review]
Looks like it does the job to me!

File: 04-refactoring_activity.Rmd (hunk 1 above)
Comment (ehumph) [reply 04-refactoring_activity.Rmd:11]: Ha, I didn't even think about that

File: _bookdown.yml (hunk 2 above)
Comment (ehumph) [reply _bookdown.yml:11]: Added!

**Pull request merged**